I can't build sources from GitHub for i386

Hello. I'm trying to compile the latest version of uwsgi for i386 processor(Intel Core i3).
Compilation always fails with this error:
`plugins/redislog/redislog_plugin.c: In function ‘uwsgi_redis_logger’:
plugins/redislog/redislog_plugin.c:162:38: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t {aka unsigned int}’ [-Werror=format=]
       setup_buf, sizeof (setup_buf), "*2\r\n$4\r\nauth\r\n$%lu\r\n%*s\r\n",
                                      ^
plugins/redislog/redislog_plugin.c:162:38: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t {aka unsigned int}’ [-Werror=format=]
plugins/redislog/redislog_plugin.c:175:38: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t {aka unsigned int}’ [-Werror=format=]
       setup_buf, sizeof (setup_buf), "*2\r\n$6\r\nselect\r\n$%lu\r\n%*s\r\n",
                                      ^
plugins/redislog/redislog_plugin.c:175:38: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t {aka unsigned int}’ [-Werror=format=]
`

Same problem on armhf which is 32 bit.
````
#12 221.5 plugins/redislog/redislog_plugin.c:162:62: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t’ {aka ‘unsigned int’} [-Werror=format=]                   
#12 221.5   162 |       setup_buf, sizeof (setup_buf), "*2\r\n$4\r\nauth\r\n$%lu\r\n%*s\r\n",
#12 221.5       |                                                            ~~^
#12 221.5       |                                                              |
#12 221.5       |                                                              long unsigned int
#12 221.5       |                                                            %u
#12 221.5   163 |       strlen(uredislog->password), (int)strlen(uredislog->password), uredislog->password);                                                                                                     
#12 221.5       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~
#12 221.5       |       |
#12 221.5       |       size_t {aka unsigned int}
#12 221.5 plugins/redislog/redislog_plugin.c:175:64: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘size_t’ {aka ‘unsigned int’} [-Werror=format=]                   
#12 221.5   175 |       setup_buf, sizeof (setup_buf), "*2\r\n$6\r\nselect\r\n$%lu\r\n%*s\r\n",
#12 221.5       |                                                              ~~^
#12 221.5       |                                                                |
#12 221.5       |                                                                long unsigned int
#12 221.5       |                                                              %u
#12 221.5   176 |              strlen(uredislog->id), (int)strlen(uredislog->id), uredislog->id);
#12 221.5       |              ~~~~~~~~~~~~~~~~~~~~~
#12 221.5       |              |
#12 221.5       |              size_t {aka unsigned int}
#12 224.9 cc1: all warnings being treated as errors
````

Link to the docker image https://github.com/SuperSandro2000/docker-images/blob/master/healthchecks/armhf.Dockerfile

--- a/plugins/redislog/redislog_plugin.c
+++ b/plugins/redislog/redislog_plugin.c
@@ -159,7 +159,7 @@ done:
 		uredislog->fd = uwsgi_connect(uredislog->address, uwsgi.socket_timeout, 0);
 		if (uredislog->password) {
 		  setup_iov.iov_len = snprintf(
-		    setup_buf, sizeof (setup_buf), "*2\r\n$4\r\nauth\r\n$%lu\r\n%*s\r\n",
+		    setup_buf, sizeof (setup_buf), "*2\r\n$4\r\nauth\r\n$%zu\r\n%*s\r\n",
 		    strlen(uredislog->password), (int)strlen(uredislog->password), uredislog->password);
 		  setup_iov.iov_base = setup_buf;
 		  ret = writev(uredislog->fd, &setup_iov, 1);
@@ -172,7 +172,7 @@ done:
 		}
 		if (uredislog->id) {
 		  setup_iov.iov_len = snprintf(
-		    setup_buf, sizeof (setup_buf), "*2\r\n$6\r\nselect\r\n$%lu\r\n%*s\r\n",
+		    setup_buf, sizeof (setup_buf), "*2\r\n$6\r\nselect\r\n$%zu\r\n%*s\r\n",
 	            strlen(uredislog->id), (int)strlen(uredislog->id), uredislog->id);
 		  setup_iov.iov_base = setup_buf;
 		  ret = writev(uredislog->fd, &setup_iov, 1);

